recipe/meta.yaml

```
@@ -1,53 +1,59 @@
-{% set version = "1.1.15" %}
+{% set version = "1.3.0" %}
 
 package:
   name: poetry
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/poetry/poetry-{{ version }}.tar.gz
-  sha256: a373848fd205f31b2f6bee6b87a201ea1e09ca573a2f40d0991539f564cedffd
+  sha256: d42943b44e80f5afafebdb696393cbdbcc750aecfaa6801174a4da42147c73fc
 
 build:
-  noarch: python
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
-    - poetry = poetry.console:main
+    - poetry = poetry.console.application:main
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
-    - poetry-core >=1.0.7,<1.1.0
+    - poetry-core >=1.1.0
   run:
     # Update version number in URL before checking dependencies
-    # https://github.com/python-poetry/poetry/blob/1.1.12/pyproject.toml
-    - python >=3.7
-    - poetry-core >=1.0.7,<1.1.0
-    - cleo >=0.8.1,<0.9.0
-    - clikit >=0.6.2,<0.7.0
-    - crashtest >=0.3.0,<0.4.0
-    - requests >=2.18,<3.0
-    - cachy >=0.3.0,<0.4.0
-    - requests-toolbelt >=0.9.1,<0.10.0
+    # https://github.com/python-poetry/poetry/blob/1.3.0/pyproject.toml
+    - python
+    - poetry-core 1.4.0
+    - poetry-plugin-export >=1.2.0,<2.0.0
+    - backports.cached-property >=1.0.2,<2.0.0  # [py<38]
     - cachecontrol >=0.12.9,<0.13.0
+    
+    # cachecontrol filecache extra
+    - filelock >=3.8.0
 
-    # cachecontrol:filecache  <https://github.com/ionrock/cachecontrol/blob/master/setup.py>
-    - lockfile >=0.9
-
-    - pkginfo >=1.4,<2.0
+    - cleo >=2.0.0,<3.0.0
+    - crashtest >=0.4.1,<0.5.0
+    - dulwich >=0.20.46,<0.21.0
+    - filelock >=3.8.0,<4.0.0
     - html5lib >=1.0,<2.0
-    - shellingham >=1.1,<2.0
-    - tomlkit >=0.7.0,<1.0.0
+    - importlib-metadata >=4.4,<5.0  # [py<310]
+    - jsonschema >=4.10.0,<5.0.0
+    - keyring >=23.9.0,<24.0.0
+    - packaging >=20.4
     - pexpect >=4.7.0,<5.0.0
-    - packaging >=20.4,<21.0
-    - virtualenv >=20.0.26,<21.0.0
-    - keyring >=21.2.0
-    - importlib-metadata >=1.6.0,<2.0.0
-
-    # Workaround for <https://github.com/conda-forge/pexpect-feedstock/issues/26>
-    - ptyprocess >=0.5
+    - pkginfo >=1.5,<2.0
+    - platformdirs >=2.5.2,<3.0.0
+    - requests >=2.18,<3.0
+    - requests-toolbelt >=0.9.1,<0.11.0
+    - shellingham >=1.5,<2.0
+    - tomli >=2.0.1,<3.0.0  # [py<311]
+    - tomlkit >=0.11.1,<1.0.0,!=0.11.2,!=0.11.3
+    - trove-classifiers >=2022.5.19
+    # See <https://github.com/python-poetry/poetry/pull/6950>
+    - virtualenv >=20.4.3,<21.0.0,!=20.4.5,!=20.4.6  # [not win or py!=39]
+    - virtualenv >=20.4.3,<20.16.6,!=20.4.5,!=20.4.6  # [win and py==39]
+    - xattr >=0.10.0,<0.11.0  # [darwin]
```

Let's document this so that we know why we aren't noarch.

```suggestion
    # xattr doesn't exist for Windows, necessitating noarch for this feedstock.
    - xattr >=0.10.0,<0.11.0  # [darwin]
```